frontend/src/main/resources/logback.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
YES! Another logger configuration file. We need this for lsp4s. Hope we can get rid of it in the future.

Reply:
No please, no Groovy. Prefer to keep it this way, I just copy pasted it from somewhere, and it's meant to be that way. We can do a whole refactoring of the loggers in a follow up PR.